commit d4ee669bcb7f934dbf73681706b7c0681b1ebd03
Author: Andy Whitcroft <apw@canonical.com>
Date:   Fri Apr 6 18:52:42 2012 +0100

    UBUNTU: SAUCE: (no-up) tools/hv: add basic manual pages
    
    BugLink: http://bugs.launchpad.net/bugs/977246
    
    Signed-off-by: Andy Whitcroft <apw@canonical.com>
    Acked-by: Leann Ogasawara <leann.ogasawara@canonical.com>
    Acked-by: Brad Figg <brad.figg@canonical.com>
    Signed-off-by: Tim Gardner <tim.gardner@canonical.com>

diff --git a/tools/hv/hv_kvp_daemon.8 b/tools/hv/hv_kvp_daemon.8
new file mode 100644
index 0000000..0fb4577
--- /dev/null
+++ b/tools/hv/hv_kvp_daemon.8
@@ -0,0 +1,26 @@
+.\"  This page Copyright (C) 2012 Andy Whitcroft <apw@canonical.com>
+.\"  Distributed under the GPL v2 or later.
+.TH HV_KVP_DAEMON 8
+.SH NAME
+hv_kvp_daemon \- Hyper-V Key Value Pair daemon
+.SH SYNOPSIS
+.ft B
+.B hv_kvp_daemon
+.br
+.SH DESCRIPTION
+\fBhv_kvp_daemon\fP
+is the userspace component of the Hyper-V key value pair functionality,
+communicating via a netlink socket with the kernel HV-KVP driver.
+This pairing allows the Hyper-V host to pass configuration information
+(such as IP addresses) to the guest and allows the host to obtain guest
+version information.
+
+.SH FILES
+.ta
+.nf
+/var/opt/hyperv/.kvp_pool_*
+.fi
+
+.SH AUTHORS
+.nf
+Written by K. Y. Srinivasan <ksrinivasan@novell.com>
